papers/david_dotson/david_dotson.rst

```diff
@@ -512,7 +513,7 @@ Although it is possible to extend ``datreant`` objects with limbs to do complex
 ``datreant`` fundamentally serves as a Pythonic interface to the filesystem, bringing value to datasets and analysis results by making them easily accessible now and later.
 As data structures and file formats change, ``datreant`` objects can always be used in the same way to supplement the way these tools are used.
 
-Because a ``Treant`` is both a Python object and a filesystem object, they work remarkably well with distributed computation libraries such as dask.distributed_ [Roc15]_ and workflow execution frameworks such as Fireworks_ [Jai15]_.
+Because each Treant is both a Python object and a filesystem object, they work remarkably well with distributed computation libraries such as dask.distributed_ [Roc15]_ and workflow execution frameworks such as Fireworks_ [Jai15]_.
```

Comment:
I've tried to use ``Treant`` when referring specifically to the Python object, and Treant anywhere else. Wasn't sure with this sentence, but I think no markup is the *more* consistent choice given that.